shared/controlflow/codeql/controlflow/Cfg.qll

```diff
@@ -1192,6 +1193,78 @@ module Make<LocationSig Location, InputSig<Location> Input> {
     }
   }
 
+  /** Provides the input to `ViewCfgQuery`. */
+  signature module ViewCfgQueryInputSig<FileSig File> {
+    /** The source file selected in the IDE. Should be an `external` predicate. */
+    string selectedSourceFile();
```

Comment:
Can we move the file, line, and column predicates to the `IdeContextual` module?

Reply:
Not sure what you mean; we can't move the `external` predicates, as they cannot be defined inside modules.